### ISSUE TITLE
Support import and export with github

### ISSUE THREAD

**Issue by jackycute** [hackmdio/hackmd#34]
Github is a huge base of markdown documents.
People might need to fetch from there and commit back, so we will try to support that.

**Comment by rlouf** [comment]
Similar to #114?

**Comment by jackycute** [comment]
Yeah, kind of duplicated issue.

**Comment by rlouf** [comment]
Yes and no. I would say #114 is a parent issue. You need to solve the problem of
pulling and extracting from a git repository first. Then, interacting with a
github repo should be straightforward---but people might want to be able to
browse their repos directly on github?

**Comment by jackycute** [comment]
I agreed, maybe would start from #114.

Below are my thoughts:
So people won't browse github repos on hackmd, I would leave it alone.
Users should put on the git url (ssh or https) which intent to associate with the note.
As a result, not only github, hackmd can integrate with any service which using git as version control.

**Comment by FranckVE** [comment]
I love HackMD that I just discovered and it is a constant wow effect on people I have invited on HackMD
I see a real interest in using HackMD to edit the documents in a GitHub repo dedicated for the technical documentation of a project : that is the starting trend in developers around me (you probably have the same feedback)

**Comment by jackycute** [comment]
Thanks @FranckVE !
I definitely think HackMD will be a great workflow for people working on document collaboratively.
So this will refer to #210 and other git related issues, we will try to support as soon as possible.

**Comment by FranckVE** [comment]
sounds outstanding ! good show @jackycute !
